addon/globalPlugins/winWizard.py

```diff
@@ -403,6 +404,7 @@ def splitToStacks(self):
 			raise RuntimeError("Cannot divide empty list into stacks")
 		else:
 			slotsToStacks = [int(str(slotNumber + 10)[:-1]) for slotNumber in takenSlots]
+			log.info(f'Stacks with hidden windows: {sorted(set(slotsToStacks))}')
```

Comment:
Please assign this to a variable - something like `stacksWithHiddenWindows` and then use it in the logging call, and in the loop below.